packages/terra-collapsible-menu-view/package.json

```
@@ -44,7 +44,7 @@
   "scripts": {
     "compile": "babel --root-mode upward src --out-dir lib --copy-files",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore --fix",
```

```suggestion
    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
```

This will potentially cause our CI to succeed but not get the updates checked in otherwise.